tests/test_forward_ref.py

```diff
@@ -44,6 +45,9 @@ def module():
         class Foo(BaseModel):
             a: 'Bar' = None
 
+            class Config:
+                undefined_types_warning = False
+
```

Comment:
@samuelcolvin @ybressler there are several pre-existing tests that needed `undefined_types_warning = False` added to their config, in order for them to work as they did previously (since the previous behavior when using a `ForwardRef` was a silent fail instead of an undefined_types_warning).  We should make sure this makes sense for all the affected tests. You both (and/or the previous test authors) are more qualified (than I am) to make that determination.

Reply:
Makes sense, I'll look.